features/plugin-activate.feature

```diff
@@ -80,3 +80,34 @@ Feature: Activate WordPress plugins
       """
       Success: No plugins installed.
       """
+
+  Scenario: Activating a plugin that does not meet PHP minimum throws a warning
+    Given a wp-content/plugins/high-requirements.php file:
+      """
+      <?php
+      /**
+       * Plugin Name: High PHP Requirements
+       * Description: This is meant to not activate because PHP version is too low.
+       * Author: WP-CLI tests
+       * Requires PHP: 99.99
+       */
+       """
+    And  I run `wp plugin deactivate --all`
+
+    When I try `wp plugin activate high-requirements`
+    Then STDERR should contain:
+      """
+      Failed to activate plugin
+      """
+    And STDERR should contain:
+      """
+      Current PHP version
+      """
+    And STDERR should contain:
+      """
+      does not meet minimum requirements
+      """
```

Comment:
Could perhaps first get the current PHP version and then use it as a placeholder for the message here?

Reply:
Resolved via https://github.com/wp-cli/extension-command/pull/253/commits/38bed6143f88827ff3a2c43a5cd3d9ce3085b7d7